api/docs/http_certificate_status.md

```diff
@@ -13,7 +13,7 @@ Find
 ----
 
     GET /puppet-ca/v1/certificate_status/:certname?environment=:environment
-    Accept: pson
+    Accept: text/pson
```

Comment:
Hmm, it worked fine when I tried from source of the latest puppetserver off of the master branch:

```
> GET /puppet-ca/v1/certificate_status/localhost HTTP/1.1
> User-Agent: curl/7.37.0
> Host: localhost:8140
> Accept: pson
>
< HTTP/1.1 200 OK
< Date: Fri, 16 Jun 2017 15:58:31 GMT
< Content-Type: pson;charset=UTF-8
< X-Puppet-Version: 5.0.0
< Vary: Accept-Encoding, User-Agent
< Content-Length: 672
* Server Jetty(9.4.4.v20170414) is not blacklisted
< Server: Jetty(9.4.4.v20170414)
<
* Connection #0 to host localhost left intact
{"name":"localhost","state":"signed","dns_alt_names":[],"fingerprint":"84:F0:06:FA:FF:D9:F7:C4:E7:83:FD:6E:27:B8:32:D9:AA:6F:15:75:0B:42:F6:1F:BF:23:CA:22:4E:12:9C:9A","fingerprints":{"SHA1":"1D:0F:29:7C:C4:07:46:88:7D:A7:FA:F3:E1:10:DE:53:C9:80:C0:6F","SHA256":"84:F0:06:FA:FF:D9:F7:C4:E7:83:FD:6E:27:B8:32:D9:AA:6F:15:75:0B:42:F6:1F:BF:23:CA:22:4E:12:9C:9A","SHA512":"4B:97:7C:9F:C3:F7:22:A6:D3:27:BD:54:2D:69:77:21:E0:60:90:8A:9C:6E:52:1F:D0:4F:38:D7:6D:7E:84:62:F3:00:94:2F:27:AE:DF:3F:3C:92:49:84:D5:24:8E:E4:B7:C5:E8:58:02:9B:78:DE:DB:D7:F2:DB:F3:1F:F4:6D","default":"84:F0:06:FA:FF:D9:F7:C4:E7:83:FD:6E:27:B8:32:D9:AA:6F:15:75:0B:42:F6:1F:BF:23:CA:22:4E:12:9C:9A"}}
```

The default auth.conf rules don't allow access to the certificate_status endpoint, so you'd have to add one in like we do for PE.  In my case, I was just testing with a wide open auth.conf from source:

```
authorization: {
    version: 1
    rules: [
        {
            match-request: {
                path: "/"
                type: "path"
            }
            allow-unauthenticated: true
            sort-order: 1
            name: "allow all"
        }
    ]
}
```